easybuild/easyblocks/c/clang.py

```diff
@@ -56,12 +56,12 @@
                  "MBlaze", "MSP430", "NVPTX", "PowerPC", "R600", "Sparc",
                  "SystemZ", "X86", "XCore"]
 
-# Mapping of EasyBuild CPU architecture names to list of default LLVM target names
+# Mapping of EasyBuild CPU architecture names to default LLVM target names
 DEFAULT_TARGETS_MAP = {
-    AARCH32: ['ARM'],
-    AARCH64: ['AArch64'],
-    POWER: ['PowerPC'],
-    X86_64: ['X86'],
+    AARCH32: 'ARM',
```

Comment:
We import `DEFAULT_TARGETS_MAP` in the LLVM easyblock, so changing the values here means trouble there...

You changed the LLVM easyblock accordingly, but there could be other easyblocks out there (outside of the central repo) that import this constant.